src/app.js

```diff
@@ -322,16 +323,21 @@ app.routeAsync("/groups/:groupName")
 app.use("/groups/:groupName/settings",
   endpoints.groups.setGroup(req => req.params.groupName));
 
+app.routeAsync("/groups/:groupName/settings")
+  .getAsync(endpoints.static.sendGatsbyEntrypoint);
+
 app.routeAsync("/groups/:groupName/settings/logo")
   .getAsync(getGroupLogo)
   .putAsync(putGroupLogo)
   .deleteAsync(deleteGroupLogo)
+  .options(optionsGroupSettings)
```

Comment:
Yeah, I wouldn't hold my breath about Express v5. ;-)

Fair point about diverging from the docs. Using `…Async()` regardless of the need for it is what I've typically been doing, so happy to keep doing that.

Reply:
Huh, I didn't add an `optionsAsync()` because this isn't async. I'll add it to keep everything uniform here.

Reply:
Fixed up in 1bf533201b6dc68fa2a9429385fbed5582d34240

Reply:
Yeah, totally reasonable to not bother with an async-safe handler registration method when the handler isn't async. That said, it's nice to default to the async-safe methods because if we add an async handler to it later without changing the method, things can get non-obviously wonky.